app/views/shelters/display.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2>All Pets for <%= @shelter.name %> Shelter</h2>
+  <%@shelter.pets.each do |pet|%>
+  <img src="<%=  pet.image %>" alt="<%= pet.image %>" style="width:200px">
+  <h4>Name: <%= pet.name %></h4>
+  <h4>Approximate Age: <%= pet.approximate_age %></h4>
+  <h4>Sex: <%= pet.sex %></h4>
+  </p>
+  <% end %>
```

Comment:
If it's only supposed to be adoptable pets being shown, might need to add a conditional statement.